addons/fix-editor-comments/addon.json

```diff
@@ -38,30 +38,35 @@
     {
       "name": "Fix dragging blocks with comments",
       "id": "fix-drag",
+      "description": "Fixes a bug where if you move a block with a comment attached to it, the comment will appear to move with it initially, but then jump back to where it was before after switching sprites or reloading.",
       "type": "boolean",
       "default": true
     },
     {
       "name": "Add scroll bars to clipped comments",
       "id": "scroll",
+      "description": "Adds scroll bars to long comments and allows you to scroll through them with the scroll wheel.",
       "type": "boolean",
       "default": true
     },
     {
       "name": "Directly edit standalone comments",
       "id": "standalone-edit",
+      "description": "Fixes a problem with comments that are not attached to blocks where if you click to edit a comment's contents, the text caret will not always get placed where you click.",
       "type": "boolean",
       "default": true
     },
     {
       "name": "Always spawn block comments next to block",
```

Comment:
```suggestion
      "name": "Always place next to block",
```

Reply:
Might want to say something different to make it clearer that this only applies to block comments, not standalone comments. Concise and understandable setting names can still be important, even with extended descriptions.

I mean, this is still alright and most importantly, short, and the setting is on by default, so maybe someone else will have a different opinion.